### ISSUE TITLE
Documentation file is not being published

### ISSUE THREAD

**Issue by nalla** [dotnet/cli#5397]
## Steps to reproduce
    mkdir test ; cd test
    dotnet.exe new
    dotnet.exe restore
    vi test.csproj

Add the following line to the property group

    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>

Then run

    dotnet.exe build
    # check bin directory for XML file
    dotnet.exe publish
    # check publish directory for XML file

## Expected  behavior
Generated XML file will be built and published.

## Actual behavior
XML file is only built.

## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.0-rc4-004553)

Product Information:
 Version:            1.0.0-rc4-004553
 Commit SHA-1 hash:  6568113e34

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\tools\dotnet.latest\sdk\1.0.0-rc4-004553
```

**Comment by livarcocc** [comment]
Please, move this to the dotnet/sdk repo.

**Comment by nalla** [comment]
Closed in favor of dotnet/sdk#695
